### ISSUE TITLE
Configuration Suggestion on Health Scaling 

### ISSUE THREAD

**Issue by Mg138** [Draylar/identity#482]
A complaint in a server I'm hosting that I'm currently seeing -- Identity is too OP.

Some complained that being able to fly fast as a bat was too good to be true, I responded that it probably makes you low on health -- which it does, but bear with me.
I logged on to test it, the health was indeed bad, but the speed was too fast, so I changed that.

But then someone mentioned "bee," I tried being a bee, and it has the same flying speed except more health, more than acceptable to be honest.

---

My thought was that I'd like to scale the health globally, e.g. multiplying health by 0.5 if you're morphing, on top of the original entity's scaling. Shouldn't be too hard to implement?

On top of that, if feasible, I'd also like to have global overrides for certain entities, e.g. one could say that as a bat you only get 1 health.

---

TLDR: global health scaling and health overrides
